apps/abbelwatch/metadata.json

```
@@ -0,0 +1,16 @@
+{ "id": "abbelwatch",
+  "name": "abbelwatch",
+  "shortName":"abbelwatch",
+  "icon": "app.png",
+  "description": "A clock that Includes a selection of fonts to personalize your watch face",
+  "type": "clock",
+  "version":"0.01",
+  "screenshots": [{"url":"screenshot.png"}],
+  "tags": "abbelwatch",
+  "readme": "README.md",
+  "supports": ["BANGLEJS2"],
+  "storage": [
+  {"name":"abbelwatch.app.js","url":"abbelwatch.app.js"},
```

```suggestion
  {"name":"abbelwatch.app.js","url":"app.js"},
```

(see corresponding comment regarding the file "abbelwatch.app.js")